### PR TITLE
🚸 Redirect user to claim page on successful payment

### PR DESCRIPTION
**Pull request by nwingt**
Before
image

After
image

### CODE REVIEW

File: src/pages/nft/fiat/stripe.vue [hunk 1]
```diff
@@ -45,7 +45,11 @@ export default {
     },
   },
   asyncData({ error, query }) {
-    const { class_id: classId, payment_id: paymentId } = query;
+    const {
+      class_id: classId,
+      payment_id: paymentId,
+      claiming_token: claimToken,
```
Comment (williamchong) [inline src/pages/nft/fiat/stripe.vue:51]: Need to change https://github.com/likecoin/likecoin-api-public/blob/master/src/routes/likernft/fiat/stripe.ts#L202 for this to work

File: src/pages/nft/fiat/stripe.vue [hunk 2]
```diff
@@ -32,20 +32,26 @@ export default {
     uiIsOpenCollectModal(isOpen) {
       if (!isOpen) {
         if (this.classId) {
-          this.$router.replace(
-            this.localeLocation({
-              name: 'nft-class-classId',
-              params: { classId: this.classId },
-            })
-          );
+          if (!(this.claimToken && this.isCompleted)) {
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/pages/nft/fiat/stripe.vue:35]: How about moving the router push from `pollForStatusUpdate` to here?

[the remark this answers is not in the document]
Comment (nwingt) [reply src/pages/nft/fiat/stripe.vue:35]: Updated

File: src/pages/nft/claim/index.vue [hunk 3]
```diff
@@ -94,6 +96,27 @@ export default {
...
     },
   },
+  watch: {
+    loginAddress: {
+      immediate: true,
+      handler() {
+        if (!this.classId || !this.token || !this.paymentId) {
+          this.state = NFT_CLAIM_STATE.MISSING_QS;
+          return;
+        }
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/pages/nft/claim/index.vue:106]: Updated

[the remark this answers is not in the document]
Comment (williamchong) [reply src/pages/nft/claim/index.vue:106]: I don't see any new commits...

**Review by williamchong — APPROVED** [review]

**Pull request merged**